src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -193,6 +194,11 @@ public class Feature
         /// </summary>
         public static Feature AggregateUnionWith => __aggregateUnionWith;
 
+        /// <summary>
+        /// Gets the allowDiskUse feature.
+        /// </summary>
+        public static AllowDiskUseFeature AllowDiskUse => __allowDiskUse;
+
```

Comment:
Rename property and move to new position in alphabetical order:

        /// <summary>
        /// Gets the find allowDiskUse feature.
        /// </summary>
        public static FindAllowDiskUseFeature FindAllowDiskUse => __findAllowDiskUse;

Reply:
Done.